pom.xml

```diff
@@ -64,21 +65,17 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.8.2</version>
+            <version>4.12</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-all</artifactId>
-            <version>1.9.0</version>
+            <version>2.0.2-beta</version>
```

Comment:
Do we need beta version? What's the benefit? Seems like a risk and need to understand the upside of using a beta release.